refractr/refractr.yml

```diff
@@ -467,262 +496,325 @@ refracts:
+  # TODO: verify this works as before
+- web.archive.org/web/20180106223649/https://webfwd.org/:
   - webfwd.org
-  - www.webfwd.org
+  #- www.webfwd.org FIXME: gives SSL Cert Error
   - webfwd.net
   - www.webfwd.net
+  status: 307
 
-# bug 1462137
-- https://hubs.mozilla.com/:
+  # bug 1462137
+- hubs.mozilla.com/:
   - hub.mozilla.com
   - hub.mozilla.org
 
-# bug 1293353
-- https://mozilla.service-now.com/: thehub.mozilla.com
+  # bug 1293353
+- mozilla.service-now.com/: thehub.mozilla.com
+  status: 302
 
-# bug 1492531
-- https://wiki.mozilla.org/Pastebin: pastebin.mozilla.org
+  # bug 1492531
+- wiki.mozilla.org/Pastebin/: pastebin.mozilla.org
 
-# bug 1492428
-- https://www.mozilla.org/en-GB/firefox/new/?utm_source=mozilla-eu&utm_medium=referral:
+  # bug 1492428
+- www.mozilla.org/en-GB/firefox/new/?utm_source=mozilla-eu&utm_medium=referral:
   - mozilla.eu
   - www.mozilla.eu
   - mozila.eu
   - www.mozila.eu
 
   # bug 1492428
-- https://www.mozilla.org/fr/firefox/new/?utm_source=mozilla-fr&utm_medium=referral:
+- www.mozilla.org/fr/firefox/new/?utm_source=mozilla-fr&utm_medium=referral:
   - mozilla.fr
   - www.mozilla.fr
 
-# bug 1463086
-- https://mozilla.org/nl/:
+  # bug 1463086
+- mozilla.org/nl/:
   - mozilla.nl
   - www.mozilla.nl
 
-# bug 1502902
-- https://events.mozilla.org/techspeakers/: techspeakers.mozilla.org
-
-  # bug 1522224
-- https://testpilot.firefox.com/: contribute.testpilot.firefox.com
+  # bug 1502902
+  #- events.mozilla.org/techspeakers/: techspeakers.mozilla.org
+- developer.mozilla.com/communities/mozilla-tech-speakers/: techspeakers.mozilla.org
 
   # bug 1533035
-- https://mozilla.github.io/geckoview/: geckoview.dev
+- mozilla.github.io/geckoview/:
+  - gv.dev
+  - geckoview.dev
+  status: 302
 
-- https://mozilla.github.io/geckoview/: gv.dev
+  # FIXME: do we need these or assume that nginx ingress will http->https?
+- developer.mozilla.com/: developer.mozilla.com
 
-- https://developer.mozilla.org/: developer.mozilla.com
+  # FIXME: do we need these or assume that nginx ingress will http->https?
+- developer.mozilla.org/: developer.mozilla.org
 
   # bug 1543423
-- https://discourse.mozilla.org/: discourse.mozilla-community.org
-  status: 307
+- discourse.mozilla.org/: discourse.mozilla-community.org
+
+- www.mozilla.org/: triagebot.mozilla.org
 
-- https://www.mozilla.org/: triagebot.mozilla.org
+  # TODO: test this works as before
+  # bug 1304806
+- phonebook.mozilla.org/: phonebook.mozilla.com
+  status: 302
 
-# TODO: test this works as before
-# bug 1304806
-- https://phonebook.mozilla.org/: phonebook.mozilla.com
-# bug 1566155
+  # bug 1566155
 - srcs: phonebook.mozilla.org
   dst:
-  - /tree.php: https://people.mozilla.org/o
-  - /*: https://people.mozilla.org/s?query=:*&who=staff
-  - /: https://people.mozilla.org/
-
-- https://pad.mozilla.org/: public.etherpad-mozilla.org
-
-# bug 1581429
-- https://support.mozilla.org/en-US/kb/canary-domain-use-application-dnsnet/: use-application-dns.net
```

Comment:
Probably because it is not active any more in prod.  Remember I did a prod vs what we had pass and made adjustments as I went.  You can use the tool itself to investigate these questions.